source/templates/stack.slim

```diff
@@ -0,0 +1,43 @@
+.tab-stack.menu-content-padding[ng-controller="StackController as stackCtrl" ng-init="stackCtrl.init()"]
+	section.cardo.stack-settings.default
+		header
+			h2 == data[:strings][:stack][:default_stack_settings][:header_title]
+			p.header-description == replaced_string(data[:strings][:stack][:default_stack_settings][:header_description], [data[:routes][:other][:bitrise_available_stacks]])
+		article
+			section
+				.max-width-wrapper
+					.stack-selector-wrapper
+						select.stack-selector[ng-model="appService.stack" ng-options="stack.name for stack in appService.availableStacks()" ng-change="appService.dockerImage = stackCtrl.defaultDockerImagesByStackIDs[stack.id] !== undefined ? '' : null"]
+					.stack-info
+						.info-badge-wrapper = svg("info_badge")
+						.stack-info-text
+							h3.stack-name[ng-bind="stackCtrl.aboutStack(appService.stack)"]
+							p.stack-description[ng-bind="appService.stack.description"]
```

Comment:
Hi @erosdome there is a missing markdown directive

Reply:
updated